setup.md

```diff
@@ -222,25 +384,9 @@ In order to preview changes locally, you must install the software described bel
     ~~~
     {: .language-bash}
 
-3.  **[Jekyll][jekyll]**.
-    You can install this by running `gem install jekyll`.
-    On macOS, a user does not have a permission to write to `/Library/Ruby/Gems/`.
-    Run `gem install jekyll --user-install` instead.
-
-4.  **R Packages**.
-    We use [knitr][cran-knitr], [stringr][cran-stringr], and [checkpoint][cran-checkpoint]
-    to format lessons written in R Markdown,
-    so you will need to install these to build R lessons
-    (and this example lesson). The best way to install these packages is to open an R terminal and type:
-
-    ~~~
-    install.packages(c('knitr', 'stringr', 'checkpoint', 'ggplot2'),
-                     repos = 'https://cran.rstudio.com', dependencies = TRUE)
-    ~~~
-    {: .language-r}
-
-If you want to run `bin/lesson_check.py` (which is invoked by `make lesson-check`)
+3. If you want to run `bin/lesson_check.py` (which is invoked by `make lesson-check`)
```

Comment:
Jekyll will be installed by bundler, only `pyyaml` will need to be installed separately, right?

Reply:
I have tested now and only pyyaml install was required after following he above steps for `make lesson-check` to run